teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative 'person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super
+    @specialization = specialization
+  end
```

Comment:
- [ ] Same here, Please add the inheritance parameters to the initialize method and super see this [example](http://ruby-for-beginners.rubymonstas.org/advanced/private_methods.html).